### PR TITLE
GAA-09 Force focus into cookie modal

### PR DESCRIPTION
**Pull request by bobmeredith**
To do (delete all that do not apply):

 - [x] Changelog entry added.

### CODE REVIEW

**Review by dahfool — APPROVED** [review]

**Review by webbyfox — APPROVED** [review]
LGTM

**Comment by webbyfox** [comment]
So new PR required on domestic-ui/international-ui for this lib upgrade.

File: react-components/src/CookiesModal.jsx [hunk 1]
```diff
@@ -7,22 +7,41 @@ import CookieManager from '../../directory_components/static/directory_component
...
+      if(focusTrap) {
+        let modal = evt.target.closest('.ReactModal__Content');
+        if(!modal || (modal == evt.target)) {
+          firstLink.focus();
+        }
+      } else {
+        setFocusTrap(true); // We can turn the trap on after the first change of focus.
+      }
+    })
+  });
```
Comment (lewis-coulson-dit) [inline react-components/src/CookiesModal.jsx:31]: I think you wanna pass an empty ` [] ` here as this code will run each render.

Comment (lewis-coulson-dit) [reply react-components/src/CookiesModal.jsx:31]: https://stackoverflow.com/questions/55565444/how-to-register-event-with-useeffect-hooks

Comment (lewis-coulson-dit) [reply react-components/src/CookiesModal.jsx:31]: Also might want to add remove event listener

**Pull request merged**